### PR TITLE
Fixed dependency issue on build.py

### PR DESCRIPTION
**Pull request by geleems**
`import polib` line in build.py is executed before running `python build.py dev_setup`
`import polib` is moved under `generate_mo()` to resolve this issue.
Some minor comments from previous PR is also covered.

### CODE REVIEW

File: mssqlcli/mssqlclioptionsparser.py [hunk 1]
```diff
@@ -120,7 +120,8 @@ def create_parser():
         dest=u'multi_subnet_failover',
         action=u'store_true',
         default=False,
-        help=u'If application is connecting to AlwaysOn AG on different subnets, setting this provides faster detection and connection to currently active server.')
+        help=u'If application is connecting to AlwaysOn AG on different subnets, setting this provides faster '
```
Comment (pensivebrian) [inline mssqlcli/mssqlclioptionsparser.py]: Let's revert this - it's already been changed and no point in changing it again.

File: mssqlcli/localized_strings.py [hunk 2]
```diff
@@ -16,4 +16,4 @@ def translation(domain=DOMAIN, localedir=LOCALE_DIR, languages=None):
 
 ## Localized Strings
 def goodbye():
-    return _(u'Goodbye!')
\ No newline at end of file
+    return _(u'Goodbye!')
```
Comment (pensivebrian) [inline mssqlcli/localized_strings.py]: Let's revert this - it's already been changed and no point in changing it again.

File: build.py [hunk 3]
```diff
@@ -6,7 +6,6 @@
 import uuid
 import platform
 import utility
-import polib
```
Comment (pensivebrian) [inline build.py]: What's the problem with having the import here?

File: mssqlcli/mssqlclioptionsparser.py (hunk 1 above)
Comment (geleems) [reply mssqlcli/mssqlclioptionsparser.py]: This is the one you asked me to revert back in previous PR, and I merged the PR before reverting back. So I reverted in this PR.

File: mssqlcli/localized_strings.py (hunk 2 above)
Comment (geleems) [reply mssqlcli/localized_strings.py]: You asked me to add newLine in previous PR. And I added it in this PR.

File: build.py (hunk 3 above)
Comment (geleems) [reply build.py]: it imports polib before installing dev dependencies.

File: mssqlcli/localized_strings.py (hunk 2 above)
Comment (geleems) [reply mssqlcli/localized_strings.py]: This is your comment in the previous PR.
image

File: mssqlcli/mssqlclioptionsparser.py (hunk 1 above)
Comment (geleems) [reply mssqlcli/mssqlclioptionsparser.py]: This is your comment in the previous PR.
image

**Comment by geleems** [comment]
Discuss offline. Discarding since not needed.

**Pull request closed without merging**